sdk/nodejs/runtime/invoke.ts

```diff
@@ -248,13 +248,13 @@ export function call<T>(tok: string, props: Inputs, res?: Resource): Output<T> {
             // Construct a provider reference from the given provider, if one is available on the resource.
             let provider: string | undefined = undefined;
             let version: string | undefined = undefined;
-            let pluginDownloadUrl: string | undefined = undefined;
+            let pluginDownloadURL: string | undefined = undefined;
```

Comment:
I renamed this just to match casing. I use the VS Code extension to show inlay hints in function calls and if the argument passed doesn't contain the name of the argument, I see a little hint that something is off:

![image](https://user-images.githubusercontent.com/788800/152037918-cacab4d5-9277-43b0-874c-473766fd421d.png)